benchmark/single-source/DictionaryRemove.swift

```diff
@@ -59,3 +65,19 @@ func removeObjects(n: Int) {
     check(dict.isEmpty)
   }
 }
+
+func filter(n: Int) {
+  for _ in 1...100*n {
+    let dict = numberMap
```

Comment:
The first invocation of `numberMap` (and `boxedNumMap` below) is going to initialize the dictionary value at runtime, which could lead to some instability here. Let's add a `setUpFunction` to the `BenchmarkInfo` declaration that simply accesses its getter to make sure that happens outside of the actual benchmarking run. Something like `setUpFunction: { blackHole(numberMap) }` ought to do the trick! (We should add this for the existing two benchmarks, too.)